packages/theming/theme-tokens/src/generated/dark-macos/reactnative/tokens-aliases.json

```diff
@@ -20,35 +20,44 @@
     "strokeWidth": 1
   },
   "brandBackground": {
-    "fillColorRest": "#185abd",
-    "fillColorHover": "#1651aa",
-    "fillColorPressed": "#0e336a",
-    "fillColorSelected": "#13458f"
+    "fillColorRest": "#0086f0",
+    "fillColorHover": "#1890f1",
+    "fillColorPressed": "#6cb8f6",
+    "fillColorDisabled": "#d6d6d6",
+    "fillColorSelected": "#3aa0f3"
```

Comment:
Curious, how do these token updates happen? Are we just copying the latest from the token repo, or does the build-tokens script do that for us?

Reply:
They get generated when I include the mappings in the input files like token-input-dark.macos.json. I added them because the macOS/iOS tokens are different from the global tokens. @rurikoaraki please correct me if I'm wrong

Reply:
Yeah currently the updates are manual so we change our input files and run the build-tokens script to regen the output if we need any changes. The fluentui-token-pipeline repo only has web tokens